client/src/components/AddClimbedBoulderForm.jsx

```diff
@@ -1,18 +1,25 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import styled from 'styled-components';
 
 const AddClimbedBoulderForm = ({
   saveClimbedBoulderToDatabase,
   formPrefilledClimbedBoulder,
 }) => {
-  const { projected, attempts, result, liked, level_feedback } =
-    formPrefilledClimbedBoulder;
+  useEffect(() => {
+    const { projected, attempts, result, liked, level_feedback } =
+      formPrefilledClimbedBoulder;
+    setNewProjected(projected);
+    setNewAttempts(attempts);
+    setNewResult(result);
+    setNewLiked(liked);
+    setNewLevelFeedback(level_feedback);
+  }, [formPrefilledClimbedBoulder]);
 
-  const [newProjected, setNewProjected] = useState(projected);
-  const [newAttempts, setNewAttempts] = useState(attempts);
-  const [newResult, setNewResult] = useState(result);
-  const [newLiked, setNewLiked] = useState(liked);
-  const [newLevelFeedback, setNewLevelFeedback] = useState(level_feedback);
+  const [newProjected, setNewProjected] = useState();
+  const [newAttempts, setNewAttempts] = useState();
+  const [newResult, setNewResult] = useState();
+  const [newLiked, setNewLiked] = useState();
+  const [newLevelFeedback, setNewLevelFeedback] = useState();
```

Comment:
A question: I think you could also work with one state and save a object with all `formPrefilledClimbedBoulder` data. 🤔 Why did you decide to do it like this?

Reply:
Ok! That is a good task for refactoring. Add it to your refactoring list and think about it later. :)

Reply:
Thank you! I have added it to the list.